specification/http/src/test/java/org/kaazing/specification/http/rfc7232/ValidatorsIT.java

```diff
@@ -37,7 +37,9 @@
 import org.kaazing.k3po.junit.rules.K3poRule;
 
 /**
- * Test to validate behavior as specified in <a href="https://tools.ietf.org/html/rfc7232">RFC 7232</a>.
+ * Test to validate behavior of <b>Etag</b> and <b>Last-Modified</b> headers as specified in
+ *  <a href="https://tools.ietf.org/html/rfc7232">RFC 7232:
+ *   Hypertext Transfer Protocol (HTTP/1.1): Conditional Requests</a>.
```

Comment:
Use regular text comments like other specifications (see WebSocket RFC-6455 specification ITs for example) rather than HTML.